src/tokens/solana.tokenlist.json

```diff
@@ -114824,10 +114824,10 @@
     {
       "chainId": 101,
       "address": "HpPJW5kyYaHomubNnF7coBNZSGrpMkZeBke3aj62i23n",
-      "symbol": "OME",
-      "name": "OneMeta",
+      "symbol": "POL",
+      "name": "Polaris",
       "decimals": 9,
-      "logoURI": "https://raw.githubusercontent.com/OneMetaLand/crypto/main/ome_logo.png",
+      "logoURI": "https://raw.githubusercontent.com/PolarisMeta/crypto/main/ome_logo.png",
```

Comment:
I renamed my GitHub account to reflect the new name of my project so needed to update the logo url.